<launch>

    <!--offsets from airsim look at json file settings-->
    <arg name="Q_param" default="0.001"/>

    <node pkg="utm" type="kf_design.py" name="kf_design">
            <param name="~Q_param" value="$(arg Q_param)"/>
    </node>

</launch>